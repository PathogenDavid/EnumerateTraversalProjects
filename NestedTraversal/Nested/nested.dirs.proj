<Project Sdk="Microsoft.Build.Traversal">
  <ItemGroup>
    <ProjectReference Include="NestedApplication/NestedApplication.csproj" />
  </ItemGroup>
</Project>
