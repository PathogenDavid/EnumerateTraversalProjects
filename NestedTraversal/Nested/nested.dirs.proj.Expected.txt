NestedTraversal/Nested/NestedApplication/NestedApplication.csproj
